### PR TITLE
1997 typography content

### LINKED ISSUE #1997: [Feature?]: [docs] Typography Docs Content
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Summary 💡

PR #1992 migrates around half of [Figma's typography doc](https://www.figma.com/proto/uJtPfI38D9i8iQg0UGK2E0/Pando-Design-Guidelines?node-id=533-15987&starting-point-node-id=6%3A11626). The remaining sections are `Color` and `Measure & Alignment`, as well as reviewing the beta docs version of Typography to make sure all is covered.

### Examples 🌈

[Figma design guidelines](https://www.figma.com/proto/uJtPfI38D9i8iQg0UGK2E0/Pando-Design-Guidelines?node-id=533-15987&starting-point-node-id=6%3A11626)

### Motivation 🔦

Unifying design & dev Typography guidelines for v1 docs

### PR DESCRIPTION
**Pull request by kathydlavoie-ps**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Closes #1997

## What is the new behavior?

* Migrates remaining typography content from [figma guidelines](https://www.figma.com/proto/uJtPfI38D9i8iQg0UGK2E0/Pando-Design-Guidelines?node-id=533-15987&starting-point-node-id=6%3A11626)
* Merges [typography doc from beta docs](https://design.pluralsight.com/docs/reference/general/typography#font-families) where helpful

## Other information

https://github.com/pluralsight/pando/assets/146388897/c7bbe677-f7d0-45e5-8c76-70fb7c631c38

### CODE REVIEW

**Review by caseybaggz — APPROVED** [review]

File: docs/src/content/docs/reference/typography.mdx [hunk 1]
```diff
@@ -178,14 +174,131 @@ All non-heading text is based on HTML elements for the utmost accessibility. Use
...
 
+### Lists
+
+We support <code>ul</code>, <code>ol</code>, and <code>dl</code> list styling.
+
+### Code Blocks
+
+For codeblocks, use the <code>code</code>, <code>kbd</code>, or <code>samp</code> tags.
+
+If you need to change the sized of the code, you can use the <code>mono</code> tokens.
```
Comment (bradryanbice) [inline docs/src/content/docs/reference/typography.mdx:185]:
```suggestion
If you need to change the size of the code, you can use the <code>mono</code> tokens.
```

File: docs/src/content/docs/reference/typography.mdx [hunk 2]
```diff
@@ -178,14 +174,131 @@ All non-heading text is based on HTML elements for the utmost accessibility. Use
...
 
+Use only text styles or variables for text colors, as labeled in the Pando Colors library.
+
+[View the Color Section](/explanation/color) for more information, descriptions, and examples of the different sentiments and colors.
+
 ## Measure & alignment
 
 ### Line-length
 
+Wrapping text for a more accessible and readable user experience. Base the line-length on the font-size, and adjust responsively.
```
Comment (bradryanbice) [inline docs/src/content/docs/reference/typography.mdx:212]:
```suggestion
Wrap text as suggested below for a more accessible and readable user experience. Base the line-length on the font-size, and adjust responsively.
```

File: docs/src/content/docs/reference/typography.mdx [hunk 3]
```diff
@@ -178,14 +174,131 @@ All non-heading text is based on HTML elements for the utmost accessibility. Use
...
+  <li>
+    <span class={css({ color: 'neutral.text.100' })}>Trailing edge align:</span>{' '}
+    Table cells containing numbers, definition list definitions/values, any
+    stacked incrementing numbers
+  </li>
+</ul>
+
+</div>
+
 ### Spacing
```
Comment (bradryanbice) [inline docs/src/content/docs/reference/typography.mdx:257]: Should we apply heading styles and type sizes here to illustrate, or will that come later?

Comment (kathydlavoie-ps) [reply docs/src/content/docs/reference/typography.mdx:252]: We can return to this in future work!

**Pull request merged**